src/swap/SwapReviewScreen.tsx

```diff
@@ -163,7 +163,7 @@ export function SwapReviewScreen() {
           />
         }
       >
-        {shouldFetch && (
+        {shouldFetch && Platform.OS === 'ios' && (
```

Comment:
hmm, would be nice to minimise platform specific code where we can. why do we have a transparent refresh control for iOS? can we remove this block, and change the `tintColor` on the `RefreshControl` to one of our grays instead?

Reply:
Changing the default Refresh control makes sense, it'll look a bit different than the mocks, but I agree that's preferable to platform specific code.

Reply:
Ended up using the default refresh control in https://github.com/valora-inc/wallet/pull/3022/commits/8410823e475b27443f5323e95e597714bf7b91f1.